platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/signed/SignedStateManager.java

```diff
@@ -140,20 +140,24 @@ public long getLastCompleteRound() {
     /**
      * Get a wrapper containing the last complete signed state.
      *
+     * @param reason a short description of why this SignedState is being reserved. Each location where a SignedState is
+     *               reserved should attempt to use a unique reason, as this makes debugging reservation bugs easier.
      * @return a wrapper with the latest complete signed state, or null if no recent states that are complete
```

Comment:
"wrapper"

Reply:
Please search this class for the term and update as needed

Reply:
done for whole class